backend/helpers/crawlerHelper.js

```diff
@@ -0,0 +1,15 @@
+module.exports = {
+  getEnabledCrawlerNames: (crawlersConfig) => {
+    const crawlersNames = [];
+
+    crawlersConfig.forEach((crawlerConfig) => {
+      if (!crawlerConfig.enabled) {
+        return;
```

Comment:
What if disabled crawler will be at first position?

Reply:
it will continue iteration